app/lib/actions/candlepin/consumer/remove_subscription.rb

```diff
@@ -0,0 +1,18 @@
+module Actions
+  module Candlepin
+    module Consumer
+      class RemoveSubscription < Candlepin::Abstract
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        input_format do
+          param :uuid, String
+          param :entitlement_id, String
```

Comment:
`entitlement_id` is different than `pool_uuid` correct? Comparing this and the attach_subscription actions